frontend/lib/src/util/utils.ts

```diff
@@ -346,7 +346,10 @@ export function setCookie(
   document.cookie = `${name}=${value};${expirationStr}path=/`
 }
 
-function isValidElementId(elementId: string): boolean {
+export function isValidElementId(elementId: string): boolean {
```

Comment:
Since you have an `!elementId` check in the function, I would change the type of the argument to `string | undefined`

Reply:
Changed 👍